.github/workflows/ci.yml

```diff
@@ -74,6 +74,10 @@ jobs:
         with:
           path: target
           key: ${{ runner.os }}-rust-${{ matrix.toolchain }}-cargo-build-target-${{ hashFiles('Cargo.lock') }}
+      - uses: actions-rs/toolchain@v1
```

Comment:
Might as well migrate to https://github.com/dtolnay/rust-toolchain while you're at it